setup.py

```diff
@@ -19,9 +19,9 @@
     url="https://github.com/quansight/qhub",
     author="Quansight",
     classifiers=[
-        "Development Status :: 3 - Alpha",
+        "07_development Status :: 3 - Alpha",
```

Comment:
Was this intentional? Looks like a find a replace issue

Reply:
Nope! I did not touch that. Some of the config I couldn't make sense of and left just as it was.